context/app/static/js/components/Header/HeaderContent/HeaderContent.jsx

```diff
@@ -28,9 +30,12 @@ function HeaderContent({ anchorRef }) {
         <>
           <FlexNoWrap>
             {['Donor', 'Sample', 'Dataset'].map((type) => (
-              <HeaderButton key={type} href={`/search?entity_type[0]=${type}`} component={Link}>
-                {`${type}s`}
-              </HeaderButton>
+              <HeaderType>
+                <StyledSvgIcon component={entityIconMap[type]} />
+                <HeaderButton key={type} href={`/search?entity_type[0]=${type}`} component={Link}>
+                  {`${type}s`}
+                </HeaderButton>
+              </HeaderType>
```

Comment:
```suggestion
              <HeaderButton
                key={type}
                href={`/search?entity_type[0]=${type}`}
                component={Link}
                startIcon={<StyledSvgIcon component={entityIconMap[type]} />}
              >
                {`${type}s`}
              </HeaderButton>
```

Reply:
There is a `startIcon` available to https://v4.mui.com/api/button/. You can then remove the `HeaderType` styles.

Reply:
That makes it much simpler, implemented the change. Double checked and it appears this is the only instance I need to make that change. Thanks!